csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "NuSpec.ReferenceGenerator": { "type": "build", "version": "2.0.0-beta-bld14" }
+  },
+  "frameworks": {
+    "netstandard1.0": {}
```

Comment:
I belive the alpha channel of Xamarin Studio does support project.json. I'll leave it to @mhutch about when support hits stable.